cmd/loadtest/main.go

```diff
@@ -11,6 +11,7 @@ import (
 	"sync"
 	"syscall"
 	"time"
+	"math"
```

Comment:
compile the project at least once before submitting a pr. go forces a specific ordering of imported libraries